rails-commenting.rb

```diff
@@ -6,25 +6,37 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) rails created all the files associated with this particular controller. From this command, the controller is used as well as the view folder. The output in the terminal would be as follows:
+        # create app/controller/blog_posts_controller.rb
+        #     invoke erb
+        #     create   app/views/main
+        #     invoke helper
+        #     create   app/helpers/main_helper.rb
+        #     invoke assets
+        #     invoke   scss
+        #     create     app/assets/stylesheets/main.scss
+
+        #  now methods can be added to the controller file that was created in the generate command. Inside the controller what the developer wants to happen is defined when the method is reached.
+
+        
 class BlogPostsController < ApplicationController
```

Comment:
What relationship is shown between BlogPostsController and ApplicationController